toonz/sources/toonz/mainwindow.cpp

```diff
@@ -1601,6 +1601,7 @@ void MainWindow::defineActions() {
   createMenuFileAction(MI_OutputSettings, tr("&Output Settings..."), "Ctrl+O");
   createMenuFileAction(MI_PreviewSettings, tr("&Preview Settings..."), "");
   createMenuFileAction(MI_Render, tr("&Render"), "Ctrl+Shift+R");
+  createMenuFileAction(MI_FastRender, tr("&Fast Render"), "Alt+R");
```

Comment:
I'm a little concerned that the term "Fast Render" may give the false impression to users that this command will render the image in a short computation time.
How about to add "MP4" to the command name (like "Fast Render to MP4" or "Quick Render to MP4") in order to clarify the role of this command?